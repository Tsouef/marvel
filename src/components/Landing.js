import React from 'react';
import Search from './Search/Search';
const Landing = () => {
  return (
    <div className="container center-align">
      <h4>Welcome to API MARVEL</h4>
      <p>Navigate with different links in menu</p>
      <Search />
    </div>
  );
};

export default Landing;
