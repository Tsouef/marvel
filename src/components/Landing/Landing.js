import React from 'react';
import SearchContainer from '../../Containers/SearchContainer';

export const Landing = () => (
  <div className="container center-align">
    <h4>Welcome to API MARVEL</h4>
    <p>Navigate with different links in menu</p>
    <SearchContainer />
  </div>
);

export default Landing;
