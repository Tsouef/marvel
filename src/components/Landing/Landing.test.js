import React from 'react';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';

import { shallow } from 'enzyme';
import { Landing } from './Landing';

describe('Landing Component', () => {
  it('renders without crashing', () => {
    const tree = shallow(<Landing />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
