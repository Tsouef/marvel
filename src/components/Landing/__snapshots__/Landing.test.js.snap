// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Component renders without crashing 1`] = `
<div
  className="container center-align"
>
  <h4>
    Welcome to API MARVEL
  </h4>
  <p>
    Navigate with different links in menu
  </p>
  <Connect(Search) />
</div>
`;
